.travis.yml

```diff
@@ -31,6 +31,11 @@ language: python
 python:
   - 2.7
 
+env:
+  - PYTEST="tests/test_views.py"
+  - PYTEST="tests/test_workflows.py"
+  - PYTEST="--ignore tests/test_views.py --ignore tests/test_workflows.py"
```

Comment:
Sure! Just for laughs, the problem of splitting them in the optimal way is NP-complete: https://en.wikipedia.org/wiki/Partition_problem, so maybe we shouldn't sweat it...